client/js/screens/Event/Event.js

```diff
@@ -28,113 +27,123 @@ const COMMENT_MUTATION = gql`
     }
   }
 `;
+class Event extends Component {
+  render() {
+    this.props.refetch();
```

Comment:
you should not be refetching manually. Use the prop from Mutation component.

Reply:
doesnt work

Reply:
didnt work